frontend/pageDesignspecs/studentPg.css

```diff
@@ -0,0 +1,13 @@
+.jumbotron2 {
+    Background-color: #ffffff;
+    margin-top: 15px;
+    padding: 20px;
+}
+p {
```

Comment:
semantic html is important -> elements should not be called "p", instead title them after what they literally are (i.e student detail text or something)